app/components/time_component.rb

```diff
@@ -0,0 +1,12 @@
+class TimeComponent < BaseComponent
+  attr_reader :time, :tag_options
+
+  def initialize(time:, **tag_options)
+    @time = time.in_time_zone.utc
```

Comment:
I'm used to seeing this syntax:
```suggestion
    @time = time.in_time_zone('UTC')
```

`.utc` is a super weird method that mutates a `Time` instance :/ :/

Reply:
> I'm used to seeing this syntax:
> 
> `.utc` is a super weird method that mutates a `Time` instance :/ :/

👍 

For context, this was added to align to the `local_time` behavior, which uses the method:

https://github.com/basecamp/local_time/blob/5edd54f5e1e79bcb8e512ba1f89077241f55fc8c/app/helpers/local_time_helper.rb#L38